### DISCUSSION TITLE
Support for Older PicoQuant Devices (PicoHarp 300)

### DISCUSSION THREAD

**Discussion by FairhallAlex** [PicoQuant/snAPI#9] [General]
Hello! My research group has a PicoHarp 300 which we have used for years and I was wondering if the snAPI has any compatibility with the older instrumentation or if that is planned at all. Is this something planned for future implementation or currently existing in another package? Alternatively, has anyone tested the PH330 library with a PH300 device?

**Comment by snizzleorg** [comment]
Unfortunately the PicoHarp300 is not supported by snAPI. Support for it is also currently not planned. The reason is that the PicoHarp300 although similar in name is actually quite different from the PH330.
